jlm/llvm/opt/push.cpp

```diff
@@ -185,7 +185,7 @@ push(jlm::rvsdg::gamma_node * gamma)
       auto argument = region->argument(n);
       for (const auto & user : *argument)
       {
-        auto tmp = input_node(user);
+        auto tmp = jlm::rvsdg::input::GetNode(*user);
```

Comment:
It seems only this file has the `jlm::` prefix included in all the replaced calls. Is there something special about this file or just a simple mistake?

Reply:
The functions are not in the jlm namespace (yet). This file has not been properly refactored yet. That is why the `jlm::` prefix was needed.